topo_processor/metadata/collection_factory.py

```diff
@@ -13,16 +13,14 @@
 def create_collection(path: str, data_type: DataType) -> Collection:
     start_time = time_in_ms()
     collection = Collection("title", "description", "license", data_type)
-    create_item(collection, path)
-    get_log().debug("CreateCollection", title=collection.title, data_type=data_type, duration=time_in_ms() - start_time)
+    create_items(collection, path)
+    get_log().debug("Collection loaded", title=collection.title, data_type=data_type, duration=time_in_ms() - start_time)
     return collection
 
 
-def create_item(collection: Collection, path: str) -> None:
-    start_time = time_in_ms()
+def create_items(collection: Collection, path: str) -> None:
     for f in os.listdir(path):
         file_path = os.path.join(path, f)
         item = Item(file_path, collection)
         repo.add_metadata(item)
         collection.items.append(item)
-    get_log().debug("CreateItem", path=path, duration=time_in_ms() - start_time)
```

Comment:
the total time to add metadata is quite a useful log to have, why was this taken out?

Reply:
I have added it back. I think I removed it by mistake.